Source/WebCore/page/PointerCaptureController.h

```diff
@@ -90,6 +90,7 @@ class PointerCaptureController {
                 ;
         }
         String pointerType;
+        RefPtr<Document> activeDocument;
```

Comment:
Should this be a `WeakPtr` instead?

Reply:
Good suggestion, pointer events shouldn't hold the document alive, I'll address this after EWS is done running tests.